pkg/sql/sem/builtins/builtins.go

```diff
@@ -2276,6 +2276,8 @@ CockroachDB supports the following flags:
 					Nanos: int64(ctx.StmtTimestamp.Sub(ctx.TxnTimestamp)),
 				}
 				if elapsed.Compare(minDuration) < 0 {
+					ctx.Txn.Proto().Restart(ctx.Txn.UserPriority(), 0, /* upgradePriority */
+						ctx.Txn.Proto().Timestamp)
```

Comment:
I guess this code is redundant with the one in `GenerateForcedRetryableError`. You can remove it?